versioned_docs/version-21.04/releases/centreon-core.md

```diff
@@ -404,6 +404,19 @@ Release date: `March 3, 2022`
 
 - Refactored the BAM Business activities downtimes inheritance mechanism so that they are properly inherited and not duplicated anymore.
 
+> You may have remaining downtimes because they were duplicated from a KPI downtime that ended before the update. In t
```

Comment:
```suggestion
> You may have remaining downtimes because they were duplicated from a KPI downtime that ended before the update. In that case, you will have to apply this procedure.
```